packages/python-packages/doc-warden/warden/cmd_entry.py

```diff
@@ -189,18 +194,32 @@ def all_operations(config):
     if config.verbose_output:
         print('Done with Changelog Content Examination')
 
-    readmes_with_wrong_case, changelogs_with_wrong_case = verify_file_case(packages, config)
-
     output_presence_results(changelog_presence_results, config)
     output_changelog_content_results(missing_changelog, empty_release_notes)
-    output_case_results(readmes_with_wrong_case, changelogs_with_wrong_case)
-
-    exit_on_readme_content_issues(readme_content_results, config)
     exit_on_changelog_content_issues(missing_changelog, empty_release_notes, config)
+    exit_on_presence_issues(changelog_presence_results, config)
 
-    if len(readme_presence_results) > 0 or len(changelog_presence_results) > 0 or len(readmes_with_wrong_case) > 0 or len(changelogs_with_wrong_case) > 0:
-        conclusion_message()
-        exit(1)
+
+# execute both presence and content verification
+def all_operations(config):
+    packages = index_packages(config)
+
+    if config.target == 'readme':
+        all_operations_readme(config)
+    elif config.target == 'changelog':
+        all_operations_changelog(config, packages)
+    elif config.target == 'all':
+        config.target_files = ['readme.rst', 'readme.md'] if config.scan_language == 'python' else ['readme.md']
```

Comment:
We should encapsulate these config options in the functions themselves. This "all" case should just be 2 lines which is operations_readme and operations_changelog